src/utils/interpret-svg.js

```diff
@@ -332,8 +358,22 @@ var getScene = function(node) {
  */
 var read = {
 
+  defsCache: new Map(),
```

Comment:
Not sure about browser compatibility here (re: #454)

Reply:
I switched to a regular object.

Reply:
Amazing, thanks.